test/Microsoft.AspNetCore.Blazor.Build.Test/BindRazorIntegrationTest.cs

```diff
@@ -18,14 +18,17 @@ public void Render_BindToComponent_SpecifiesValue_WithMatchingProperties()
             // Arrange
             AdditionalSyntaxTrees.Add(Parse(@"
 using System;
+using Microsoft.AspNetCore.Blazor;
 using Microsoft.AspNetCore.Blazor.Components;
 
 namespace Test
 {
     public class MyComponent : BlazorComponent
     {
+        [Parameter]
```

Comment:
What do you think about maybe putting the new attribute into the .Components namespace? It’s only usable in the context where you are also using IComponent or BlazorComponent. Also would avoid all these extra usings.

Reply:
I don’t mind hugely BTW.

Reply:
Yeah I think this is fine. We might want to consider consolidating all the things we expect 99% of users to use into a single namespace but your suggesting is in line with the current design so let's do that.